spec/features/merchants/invoices/show_spec.rb

```diff
@@ -64,12 +64,26 @@
       expect(page).to have_content(@invoice_1.status)
       expect(page).to_not have_content(@item_7.name)
     end
-  end
 
-  it "Shows total revenue generated for an invoice" do
-    visit merchant_invoice_path(@merchant_1, @invoice_1)
+    it "Shows total revenue generated for an invoice" do
+      visit merchant_invoice_path(@merchant_1, @invoice_1)
+
+      expect(page).to have_content("Total revenue: $1,000.00")
 
-    expect(page).to have_content("Total revenue: $1,000.00")
+    end
+  end
 
+  #US 18
+  describe "Merchant Invoice Show Page: Update Item Status" do
```

Comment:
Testing needs to show the current status is selected and options to select a new status to update the attribute. US: I see that each invoice item status is a select field
And I see that the invoice item's current status is selected
When I click this select field,
Then I can select a new status for the Item